demos/tutorials/6_fluid_tank.ipynb

```diff
@@ -0,0 +1,280 @@
+{
+  "nbformat": 4,
+  "nbformat_minor": 0,
+  "metadata": {
+    "colab": {
+      "provenance": []
+    },
+    "kernelspec": {
+      "name": "python3",
+      "display_name": "Python 3"
+    },
+    "language_info": {
+      "name": "python"
+    }
+  },
+  "cells": [
+    {
+      "cell_type": "markdown",
+      "source": [
+        "# Introduction\n",
```

Comment:
This should provide a more descriptive name of what we will do:
Suggestion: "Optimal Geometry Design of a Fluid Tank"

Reply:
I would also not focus so much on optimization, since that points output to find the "best" solution from all feasible solutions.